pkgs/development/python-modules/bugwarrior/default.nix

```diff
@@ -1,31 +1,32 @@
-{ stdenv, buildPythonPackage, fetchPypi
-, mock, unittest2, nose
+{ stdenv, buildPythonPackage, fetchPypi, pythonOlder
+# , mock, unittest2, nose
```

Comment:
```suggestion
, mock, unittest2, nose
```

Reply:
I have squashed and just removed the commented out test deps.